src/stubgen.py

```diff
@@ -312,8 +364,9 @@ def put_function(self, fn, name: Optional[str] = None, parent=None):
 
         if name is None:
             name = fn.__name__
+            assert name
 
-        overloads: Sequence[Callable] = []
+        overloads: Sequence[Callable[..., Any]] = []
```

Comment:
Stupid question, isn't `Callable` and `Callable[..., Any]` the same? I don't see the latter type adding any useful extra information.

Reply:
See also this.